src/plugins/manus/install_manus.sh

```
@@ -0,0 +1,161 @@
+# Find TeleopCore root (3 levels up from src/plugins/manus)
+TELEOP_ROOT="$(cd "$SCRIPT_DIR/../../.." && pwd)"
+cd "$TELEOP_ROOT"
+
+echo "TeleopCore root: $TELEOP_ROOT"
+
+echo "Configuring CMake..."
+cmake -S . -B build -DCMAKE_BUILD_TYPE=Release
+
+# Build the plugin and the diagnostic printer tool
+echo "Building..."
+cmake --build build --target manus_hand_plugin manus_hand_tracker_printer -j$(nproc)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)